widgetastic_manageiq/__init__.py

```diff
@@ -3660,7 +3660,31 @@ def fill(self, value):
         return True
 
 
-class PotentiallyInvisibleTab(Tab):
+class WaitTab(Tab):
+    def child_widget_accessed(self, widget):
+        _to_try = 2
+        _tries = 0
+        while _tries < _to_try:
```

Comment:
2 attempts ? you mentioned 3 when you talked about it.

Reply:
Just two tries, don't want it taking up more time than that for workaround